doc/api/deprecations.md

```diff
@@ -818,6 +818,14 @@ a future version at which point only authentication tag lengths of 128, 120,
 is not included in this list will be considered invalid in compliance with
 [NIST SP 800-38D][].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: process.binding('util').is[...] typechecks
+
+Type: Documentation-only/`--pending-deprecation`
```

Comment:
Is `--pending-deprecation` the correct type here? It's the first such one, afaik.
/cc @jasnell, I suppose.

Also, `--pending-deprecation` (and it being a `semver-minor`) is not mentioned in [COLLABORATOR_GUIDE.md#deprecations](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#deprecations), but that's probably a separate issue.

Reply:
I opened #18417 for that.

Reply:
> As I understand, currently `--pending-deprecation` is just a subclass of doc-only deprecations.

I’m okay with seeing it that way, yeah. And I don’t mind being explicit about this here.

Reply:
#18433 landed, so could you change this line to 
```
Type: Documentation-only (supports [`--pending-deprecation`][])
```
for consisntency?